src/theme/theme.ts

```diff
@@ -43,6 +41,12 @@ const LightMaterialUITheme = createTheme(theme, {
                 height: sizesConfig.buttonPrimary.height,
             },
         },
+        toolbar: {
```

Comment:
Where are these styles used?

Reply:
I plan to use them to style icon toolbar, currently it's suppose to be a placeholder.